packages/framework-core/src/services/graphql/graphql-query-generator.ts

```diff
@@ -78,6 +79,26 @@ export class GraphQLQueryGenerator {
     return queries
   }
 
+  private generateFilterListedQueries(): GraphQLFieldConfigMap<unknown, GraphQLResolverContext> {
```

Comment:
Nitpicking. What about calling this `generateListedQueries`? It is shorter and we can infer any list can be filtered 😊